src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/IconUrlToImageCacheConverter.cs

```diff
@@ -24,15 +31,15 @@ internal class IconUrlToImageCacheConverter : IValueConverter
             WebExceptionStatus.UnknownError
         };
 
-        private static readonly System.Net.Cache.RequestCachePolicy RequestCacheIfAvailable = new System.Net.Cache.RequestCachePolicy(System.Net.Cache.RequestCacheLevel.CacheIfAvailable);
+        private static readonly RequestCachePolicy RequestCacheIfAvailable = new RequestCachePolicy(RequestCacheLevel.CacheIfAvailable);
 
         private static readonly ErrorFloodGate _errorFloodGate = new ErrorFloodGate();
 
         // We bind to a BitmapImage instead of a Uri so that we can control the decode size, since we are displaying 32x32 images, while many of the images are 128x128 or larger.
         // This leads to a memory savings.
-        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
         {
-            var iconUrl = value as Uri;
+            var iconUrl = values[0] as Uri;
```

Comment:
Nice catch! I just assumed that the two values always will be there, because those are specified in the XAML style

https://github.com/NuGet/NuGet.Client/blob/ca22f3c30e619167b34fd84a152bd5a94aaa8d63/src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Resources.xaml#L697